crates/lib/src/install.rs

```diff
@@ -2235,9 +2235,15 @@ pub(crate) async fn install_to_filesystem(
             Some(MountSpec::new(&spec, "/boot"))
         }
     } else {
-        boot_uuid
-            .as_deref()
-            .map(|boot_uuid| MountSpec::new_uuid_src(boot_uuid, "/boot"))
+        // Read /etc/fstab to get boot entry, but only use it if it's UUID-based
```

Comment:
Hum...I think we can unconditionally propagate this; I don't think we need to filter to UUID only.